ui/src/app/machines/views/MachineDetails/MachineStorage/AvailableStorageTable/AddLogicalVolume/AddLogicalVolume.test.tsx

```diff
@@ -72,6 +73,93 @@ describe("AddLogicalVolume", () => {
     expect(wrapper.find("Input[name='name']").prop("value")).toBe("lv2");
   });
 
+  it("can validate if the size meets the minimum requirement", async () => {
+    const disk = diskFactory({
+      available_size: 1000000000, // 1GB
+      id: 1,
+      type: DiskTypes.VOLUME_GROUP,
+    });
+    const state = rootStateFactory({
+      machine: machineStateFactory({
+        items: [machineDetailsFactory({ disks: [disk], system_id: "abc123" })],
+        statuses: machineStatusesFactory({
+          abc123: machineStatusFactory(),
+        }),
+      }),
+    });
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <AddLogicalVolume
+          closeExpanded={jest.fn()}
+          disk={disk}
+          systemId="abc123"
+        />
+      </Provider>
+    );
+
+    // Set logical volume size to 0.1MB
+    await act(async () => {
+      wrapper
+        .find("input[name='size']")
+        .props()
+        .onChange({
+          target: { name: "size", value: "0.1" },
+        } as React.ChangeEvent<HTMLInputElement>);
+    });
+    wrapper.update();
+    await act(async () => {
+      wrapper
+        .find("select[name='unit']")
+        .props()
+        .onChange({
+          target: { name: "unit", value: "MB" },
+        } as React.ChangeEvent<HTMLSelectElement>);
+    });
+    wrapper.update();
+
+    expect(wrapper.find(".p-form-validation__message").exists()).toBe(true);
```

Comment:
I think these test should check for the string (or part of the string) as they could be showing a different validation message to the one you want.

Reply:
You're right! Updating them all now